modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/notifications/ContactsCenterUserNotificationHandler.java

```diff
@@ -108,6 +102,16 @@ protected String getBody(
 				new Object[] {creatorUserName});
 		}
 
+		if ((socialRequest.getStatus() !=
+				SocialRequestConstants.STATUS_PENDING) ||
+			(socialRequest.getCreateDate() >
+				userNotificationEvent.getTimestamp())) {
```

Comment:
Oh, man. Updating a create date is definitely not a good idea. I've seen how it's done in SocialRequestLocalServiceImpl.jave:87 but I consider this to be a bug that should be fixed sooner or later.

It seems that using modifiedDate would also be correct and more future-proof. Do you agree?